File: projects/calculator/java/Calculator.java
import java.util.Scanner;

public class Calculator {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Simple Calculator");
        System.out.println("1. Add  2. Subtract  3. Multiply  4. Divide");
        System.out.print("Choice: ");
        int choice = sc.nextInt();

        System.out.print("First number: ");
        double a = sc.nextDouble();
        System.out.print("Second number: ");
        double b = sc.nextDouble();

        switch (choice) {
            case 1 -> System.out.println("Result: " + (a + b));
            case 2 -> System.out.println("Result: " + (a - b));
            case 3 -> System.out.println("Result: " + (a * b));
            case 4 -> {
                if (b != 0) System.out.println("Result: " + (a / b));
                else System.out.println("Cannot divide by zero");
            }
            default -> System.out.println("Invalid");
        }
        sc.close();
    }
}

File: projects/calculator/php/calculator.php
<?php
echo "Simple Calculator\n";
echo "1. Add  2. Subtract  3. Multiply  4. Divide\n";
echo "Choice: ";
$choice = trim(fgets(STDIN));
echo "First number: ";
$a = (float)trim(fgets(STDIN));
echo "Second number: ";
$b = (float)trim(fgets(STDIN));

switch ($choice) {
    case "1": echo "Result: " . ($a + $b) . "\n"; break;
    case "2": echo "Result: " . ($a - $b) . "\n"; break;
    case "3": echo "Result: " . ($a * $b) . "\n"; break;
    case "4":
        if ($b != 0) echo "Result: " . ($a / $b) . "\n";
        else echo "Cannot divide by zero\n";
        break;
    default: echo "Invalid\n";
}
